src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/CftQueryServiceClaimTaskTest.java

```diff
@@ -0,0 +1,297 @@
+        Assertions.assertThat(task.isEmpty()).isTrue();
+    }
+
+    @Test
+    void should_return_empty_task_resource_when_authorisations_are_wrong_for_challenge_grant_type() {
+        final String taskId = "8d6cc5cf-c973-11eb-bdba-0242ac111018";
+        final String caseId = "1623278362431018";
+        List<RoleAssignment> roleAssignments = new ArrayList<>();
+        final Map<String, String> tcAttributes = Map.of(
+            RoleAttributeDefinition.CASE_TYPE.value(), "Asylum",
+            RoleAttributeDefinition.JURISDICTION.value(), "IA",
+            RoleAttributeDefinition.CASE_ID.value(), caseId
+        );
+        RoleAssignment roleAssignment = RoleAssignment.builder().roleName("tribunal-caseworker")
+            .classification(Classification.PUBLIC)
+            .beginTime(LocalDateTime.now().minusYears(1))
+            .endTime(LocalDateTime.now().plusYears(1))
+            .authorisations(List.of("PROBATE","SCSS"))
+            .grantType(GrantType.CHALLENGED)
+            .attributes(tcAttributes)
+            .build();
+        roleAssignments.add(roleAssignment);
+
+        AccessControlResponse accessControlResponse = new AccessControlResponse(null, roleAssignments);
+        permissionsRequired.add(PermissionTypes.OWN);
+        permissionsRequired.add(PermissionTypes.EXECUTE);
+
+        final Optional<TaskResource> task = cftQueryService.getTask(taskId, accessControlResponse, permissionsRequired);
+        Assertions.assertThat(task.isEmpty()).isTrue();
+    }
+
+    @Test
+    void should_return_empty_task_resource_when_authorisations_is_empty_for_challenge_grant_type() {
+        final String taskId = "8d6cc5cf-c973-11eb-bdba-0242ac111018";
+        final String caseId = "1623278362431018";
+        List<RoleAssignment> roleAssignments = new ArrayList<>();
+        final Map<String, String> tcAttributes = Map.of(
+            RoleAttributeDefinition.CASE_TYPE.value(), "Asylum",
+            RoleAttributeDefinition.JURISDICTION.value(), "IA",
+            RoleAttributeDefinition.CASE_ID.value(), caseId
+        );
+        RoleAssignment roleAssignment = RoleAssignment.builder().roleName("tribunal-caseworker")
+            .classification(Classification.PUBLIC)
+            .beginTime(LocalDateTime.now().minusYears(1))
+            .endTime(LocalDateTime.now().plusYears(1))
+            .grantType(GrantType.CHALLENGED)
+            .attributes(tcAttributes)
+            .build();
+        roleAssignments.add(roleAssignment);
+
+        AccessControlResponse accessControlResponse = new AccessControlResponse(null, roleAssignments);
+        permissionsRequired.add(PermissionTypes.OWN);
+        permissionsRequired.add(PermissionTypes.EXECUTE);
+
+        final Optional<TaskResource> task = cftQueryService.getTask(taskId, accessControlResponse, permissionsRequired);
+        Assertions.assertThat(task.isEmpty()).isTrue();
+    }
+
+}
```

Comment:
should we cover Standard & Specific grantTypes as well? i.e. permissions wrong & missing scenarios